packages/api/src/hooks/useCtraderAccountsList.ts

```diff
@@ -25,6 +26,10 @@ const useCtraderAccountsList = () => {
                     ...account,
                     /** Account's currency config information */
                     currency_config: account.currency ? getConfig(account.currency) : undefined,
+                    /** The balance of the account in currency format. */
+                    display_balance: displayMoney(account?.balance || 0, account?.currency || 'USD', {
```

Comment:
@sergei-deriv delete this and use `formatted_balance` instead to prevent overwrite of `display_balance` from the BE